crates/test_utils/Cargo.toml

```diff
@@ -17,8 +17,9 @@
 
 [package]
 name = "iceberg_test_utils"
-version = "0.1.0"
-edition = "2021"
+version = {workspace = true} 
+edition = {workspace = true} 
+homepage = {workspace = true} 
 
```

Comment:
I think we also need repository and license for this crate?

Reply:
done!